superset/common/query_object.py

```diff
@@ -66,8 +66,8 @@ def __init__(
         extras: Optional[Dict] = None,
         columns: Optional[List[str]] = None,
         orderby: Optional[List[List]] = None,
-        relative_start: str = app.config.get("DEFAULT_RELATIVE_START_TIME", "today"),
-        relative_end: str = app.config.get("DEFAULT_RELATIVE_END_TIME", "today"),
+        relative_start: str = app.config["DEFAULT_RELATIVE_START_TIME"],
+        relative_end: str = app.config["DEFAULT_RELATIVE_END_TIME"],
```

Comment:
This is a good example of why this change is good; no point in having secondary default values spread across the codebase, as these should only be defaulted in one place.